src/routes/accounts.py

```
@@ -254,6 +259,76 @@
+            if expires_at > datetime.now(timezone.utc):
+                raise HTTPException(
+                    status_code=status.HTTP_409_CONFLICT,
+                    detail="User already have an activation link.",
+                )
+            else:
+                await db.execute(
+                    delete(ActivationTokenModel).where(
+                        ActivationTokenModel.id == user.activation_token.id
+                    )
+                )
+                await db.flush()
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "raise", remove the "else" and de-indent the code inside it

(R1720)